.github/ISSUE_TEMPLATE/docs_or_website.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Documentation/Website Bug Report
+about: For bugs involving the Cosmos documentation & website
+title: ''
+labels: Bug
```

Comment:
I dont think this should have the Bug label, maybe add a new one for documentation?

Reply:
I just remembered there's the (Area: Documentation) label which will be just fine for this purpose :)